Neos.Eel/Classes/Helper/ArrayHelper.php

```diff
@@ -366,6 +366,20 @@ public function range($start, $end, $step = 1)
         return range($start, $end, $step);
     }
 
+    /**
+     * Set the specified key in the the array
+     *
+     * @param array $array
+     * @param mixed $key the Key that will be set
```

Comment:
doesn't the key have to be a `string`?

Reply:
At least integer would be ok aswell

Reply:
@bwaidelich i altered the annotation to `string|integer`